src/tiles/constants.ts

```diff
@@ -1 +1,18 @@
+import { TileMatrixSet } from './tileMatrixSet';
+import { TILEMATRIXSETJSON_WEB_MERCATOR_QUAD } from './tileMatrixSets/webMercatorQuad';
+import { TILEMATRIXSETJSON_WORLD_CRS84_QUAD } from './tileMatrixSets/worldCRS84Quad';
+
 export const cornerOfOriginCode = ['topLeft', 'bottomLeft'] as const;
+
+/**
+ * Tile Matrix Set World CRS84 Quad, This Tile Matrix Set defines tiles in the Equirectangular Plate Carrée projection in the CRS84 CRS for the whole world - https://docs.ogc.org/is/17-083r4/17-083r4.html#toc50
+ * @category Tile Matrix Set
+ */
+export const TILEMATRIXSET_WORLD_CRS84_QUAD = new TileMatrixSet(TILEMATRIXSETJSON_WORLD_CRS84_QUAD);
+
+/**
+ * Tile Matrix Set Web Mercator Quad, This Tile Matrix Set defines tiles based on a spherical Mercator instead of an ellipsoid - https://docs.ogc.org/is/17-083r4/17-083r4.html#toc49
+ * @
+ * @category Tile Matrix Set
+ */
+export const TILEMATRIXSET_WEB_MERCATOR_QUAD = new TileMatrixSet(TILEMATRIXSETJSON_WEB_MERCATOR_QUAD);
```

Comment:
most of `"tile"` calculations ican think of will be correct for most of tiles matrix sets. transformations will might include crs transformations which will be complex to implement and can have runtime computational burden

Reply:
Of course it will cost us in performance, but if we support this then we should at least throw errors when getting a `TileMatrixSet` that isn't `WORLD_CRS84_QUAD` so the user won't get unexpected behavior. Also, maybe add it to the README (in the future when it will exist) that currently we only support calculations for `WORLD_CRS84_QUAD`.

Reply:
where did you noticed an unexpected bahavior?
code concerning `tiles` should be invariant to the `TileMatrixSet` itself
so it doesn't matter if you called `WORLD_CRS84_QUAD` or `WEB_MERCATOR_QUAD` or anything else

Reply:
The unexpected behavior comes when I make calculations on tiles compared to a certain `TileMatrixSet`, or any other calculations. I guess now with validations we will throw an error because they don't match, but it should be possible in general.